### ISSUE TITLE
Oh my zsh suspend the execution after update

### ISSUE THREAD

**Issue by rsignavong** [r-darwish/topgrade#575]
**What did you expect to happen?**
...
The execution should continue to the next upgrade step.

**What actually happened?**
...
The execution is suspended and don't continue even after an `fg` command after suspension.

**Additional details:**
- Which operating system or Linux distribution are you using?
MacOS Big Sur
- How did you install Topgrade?
Topgrade is installed with Cargo
The current version is v6.0.1
- If a specific step is failing, what happens when you run the command on your own? (Run `topgrade --dry-run` to see which commands Topgrade is running)
```
―― 12:13:14 - oh-my-zsh ――――――――――――――――――――――――――――――――――――――――――――――――――――――――
Pulling custom plugins and themes
Would pull /Users/rockysignavong/.oh-my-zsh/custom/themes
Dry running: zsh /Users/rockysignavong/.oh-my-zsh/tools/upgrade.sh
```
Every steps return `OK`, even oh-my-zsh

<details><pre>
―― 12:07:05 - oh-my-zsh ――――――――――――――――――――――――――――――――――――――――――――――――――――――――
 2020-12-04T11:07:05.251Z TRACE topgrade::executor   > Output of "zsh" "-c" "test $ZSH_CUSTOM && echo -n $ZSH_CUSTOM": Output { status: ExitStatus(ExitStatus(256)), stdout: "", stderr: "" }
 2020-12-04T11:07:05.251Z DEBUG topgrade::steps::zsh > Running zsh returned exit code: 1: . Using default path: /Users/rockysignavong/.oh-my-zsh/custom
 2020-12-04T11:07:05.251Z DEBUG topgrade::steps::zsh > oh-my-zsh custom dir: /Users/rockysignavong/.oh-my-zsh/custom
 2020-12-04T11:07:05.251Z DEBUG topgrade::steps::git > Checking if /Users/rockysignavong/.oh-my-zsh/custom is a git repository
 2020-12-04T11:07:05.260Z TRACE topgrade::executor   > Output of "/usr/local/bin/git" "rev-parse" "--show-toplevel": Output { status: ExitStatus(ExitStatus(0)), stdout: "/Users/rockysignavong/.oh-my-zsh\n", stderr: "" }
 2020-12-04T11:07:05.260Z DEBUG topgrade::steps::git > /Users/rockysignavong/.oh-my-zsh/custom/example.zsh is a file. Checking /Users/rockysignavong/.oh-my-zsh/custom
 2020-12-04T11:07:05.260Z DEBUG topgrade::steps::git > Checking if /Users/rockysignavong/.oh-my-zsh/custom is a git repository
 2020-12-04T11:07:05.271Z TRACE topgrade::executor   > Output of "/usr/local/bin/git" "rev-parse" "--show-toplevel": Output { status: ExitStatus(ExitStatus(0)), stdout: "/Users/rockysignavong/.oh-my-zsh\n", stderr: "" }
 2020-12-04T11:07:05.272Z DEBUG topgrade::steps::git > Checking if /Users/rockysignavong/.oh-my-zsh/custom/themes is a git repository
 2020-12-04T11:07:05.281Z TRACE topgrade::executor   > Output of "/usr/local/bin/git" "rev-parse" "--show-toplevel": Output { status: ExitStatus(ExitStatus(0)), stdout: "/Users/rockysignavong/.oh-my-zsh/custom/themes\n", stderr: "" }
 2020-12-04T11:07:05.281Z DEBUG topgrade::steps::git > /Users/rockysignavong/.oh-my-zsh/custom/themes/LICENSE is a file. Checking /Users/rockysignavong/.oh-my-zsh/custom/themes
 2020-12-04T11:07:05.281Z DEBUG topgrade::steps::git > Checking if /Users/rockysignavong/.oh-my-zsh/custom/themes is a git repository
 2020-12-04T11:07:05.287Z TRACE topgrade::executor   > Output of "/usr/local/bin/git" "rev-parse" "--show-toplevel": Output { status: ExitStatus(ExitStatus(0)), stdout: "/Users/rockysignavong/.oh-my-zsh/custom/themes\n", stderr: "" }
 2020-12-04T11:07:05.287Z DEBUG topgrade::steps::git > /Users/rockysignavong/.oh-my-zsh/custom/themes/example.zsh-theme is a file. Checking /Users/rockysignavong/.oh-my-zsh/custom/themes
 2020-12-04T11:07:05.287Z DEBUG topgrade::steps::git > Checking if /Users/rockysignavong/.oh-my-zsh/custom/themes is a git repository
 2020-12-04T11:07:05.297Z TRACE topgrade::executor   > Output of "/usr/local/bin/git" "rev-parse" "--show-toplevel": Output { status: ExitStatus(ExitStatus(0)), stdout: "/Users/rockysignavong/.oh-my-zsh/custom/themes\n", stderr: "" }
 2020-12-04T11:07:05.297Z DEBUG topgrade::steps::git > /Users/rockysignavong/.oh-my-zsh/custom/themes/makefile is a file. Checking /Users/rockysignavong/.oh-my-zsh/custom/themes
 2020-12-04T11:07:05.297Z DEBUG topgrade::steps::git > Checking if /Users/rockysignavong/.oh-my-zsh/custom/themes is a git repository
 2020-12-04T11:07:05.304Z TRACE topgrade::executor   > Output of "/usr/local/bin/git" "rev-parse" "--show-toplevel": Output { status: ExitStatus(ExitStatus(0)), stdout: "/Users/rockysignavong/.oh-my-zsh/custom/themes\n", stderr: "" }
 2020-12-04T11:07:05.305Z DEBUG topgrade::steps::git > /Users/rockysignavong/.oh-my-zsh/custom/themes/bullet-train.zsh-theme is a file. Checking /Users/rockysignavong/.oh-my-zsh/custom/themes
 2020-12-04T11:07:05.305Z DEBUG topgrade::steps::git > Checking if /Users/rockysignavong/.oh-my-zsh/custom/themes is a git repository
 2020-12-04T11:07:05.314Z TRACE topgrade::executor   > Output of "/usr/local/bin/git" "rev-parse" "--show-toplevel": Output { status: ExitStatus(ExitStatus(0)), stdout: "/Users/rockysignavong/.oh-my-zsh/custom/themes\n", stderr: "" }
 2020-12-04T11:07:05.314Z DEBUG topgrade::steps::git > /Users/rockysignavong/.oh-my-zsh/custom/themes/README.md is a file. Checking /Users/rockysignavong/.oh-my-zsh/custom/themes
 2020-12-04T11:07:05.314Z DEBUG topgrade::steps::git > Checking if /Users/rockysignavong/.oh-my-zsh/custom/themes is a git repository
 2020-12-04T11:07:05.320Z TRACE topgrade::executor   > Output of "/usr/local/bin/git" "rev-parse" "--show-toplevel": Output { status: ExitStatus(ExitStatus(0)), stdout: "/Users/rockysignavong/.oh-my-zsh/custom/themes\n", stderr: "" }
 2020-12-04T11:07:05.321Z DEBUG topgrade::steps::git > Checking if /Users/rockysignavong/.oh-my-zsh/custom/themes/img is a git repository
 2020-12-04T11:07:05.331Z TRACE topgrade::executor   > Output of "/usr/local/bin/git" "rev-parse" "--show-toplevel": Output { status: ExitStatus(ExitStatus(0)), stdout: "/Users/rockysignavong/.oh-my-zsh/custom/themes\n", stderr: "" }
 2020-12-04T11:07:05.331Z DEBUG topgrade::steps::git > /Users/rockysignavong/.oh-my-zsh/custom/themes/.gitignore is a file. Checking /Users/rockysignavong/.oh-my-zsh/custom/themes
 2020-12-04T11:07:05.331Z DEBUG topgrade::steps::git > Checking if /Users/rockysignavong/.oh-my-zsh/custom/themes is a git repository
 2020-12-04T11:07:05.341Z TRACE topgrade::executor   > Output of "/usr/local/bin/git" "rev-parse" "--show-toplevel": Output { status: ExitStatus(ExitStatus(0)), stdout: "/Users/rockysignavong/.oh-my-zsh/custom/themes\n", stderr: "" }
 2020-12-04T11:07:05.342Z DEBUG topgrade::steps::git > Checking if /Users/rockysignavong/.oh-my-zsh/custom/themes/.git is a git repository
 2020-12-04T11:07:05.348Z TRACE topgrade::executor   > Output of "/usr/local/bin/git" "rev-parse" "--show-toplevel": Output { status: ExitStatus(ExitStatus(32768)), stdout: "", stderr: "fatal: this operation must be run in a work tree\n" }
Pulling custom plugins and themes
 2020-12-04T11:07:05.356Z TRACE topgrade::executor   > Output of "/usr/local/bin/git" "remote" "show": Output { status: ExitStatus(ExitStatus(0)), stdout: "origin\n", stderr: "" }
 2020-12-04T11:07:05.364Z TRACE topgrade::executor   > Output of "/usr/local/bin/git" "rev-parse" "HEAD": Output { status: ExitStatus(ExitStatus(0)), stdout: "d60f62c34b3d9253292eb8be81fb46fa65d8f048\n", stderr: "" }
Pulling /Users/rockysignavong/.oh-my-zsh/custom/themes
 2020-12-04T11:07:06.689Z TRACE topgrade::executor   > Output of "/usr/local/bin/git" "rev-parse" "HEAD": Output { status: ExitStatus(ExitStatus(0)), stdout: "d60f62c34b3d9253292eb8be81fb46fa65d8f048\n", stderr: "" }
Up-to-date /Users/rockysignavong/.oh-my-zsh/custom/themes
 2020-12-04T11:07:06.689Z DEBUG topgrade::executor   > Running "zsh" "/Users/rockysignavong/.oh-my-zsh/tools/upgrade.sh"
Updating Oh My Zsh
From https://github.com/ohmyzsh/ohmyzsh
 * branch              master     -> FETCH_HEAD
Current branch master is up to date.
         __                                     __
  ____  / /_     ____ ___  __  __   ____  _____/ /_
 / __ \/ __ \   / __ `__ \/ / / /  /_  / / ___/ __ \
/ /_/ / / / /  / / / / / / /_/ /    / /_(__  ) / / /
\____/_/ /_/  /_/ /_/ /_/\__, /    /___/____/_/ /_/
                        /____/

Oh My Zsh is already at the latest version.
To keep up with the latest news and updates, follow us on Twitter: https://twitter.com/ohmyzsh
Want to get involved in the community? Join our Discord: https://discord.gg/ohmyzsh
Get your Oh My Zsh swag at: https://shop.planetargon.com/collections/oh-my-zsh
 2020-12-04T11:07:08.181Z DEBUG topgrade::terminal   > Desktop notification: oh-my-zsh failed

Retry? (y)es/(N)o/(s)hell[1]  + 65835 suspended (tty output)  topgrade -v
</pre></details>

**Comment by miterion** [comment]
Had the same problem but than I saw that this was fixed in f9116dd0f3a33f40e739498d83a7207198a94b26. It will probably be in the next release.

**Comment by r-darwish** [comment]
This is probably a combination of #475 and #574
